src/test/java/com/lob/client/test/AddressTest.java

```diff
@@ -27,6 +27,9 @@
     @Test
     public void testListAddresses() throws Exception {
         final AddressResponseList addresses = client.getAddresses().get();
+        assertEquals(200, addresses.getStatusCode());
```

Comment:
Added another quick commit fixing all assertEquals. Figured it could go in with this PR.